cmd/frontend/graphqlbackend/search.go

```diff
@@ -754,12 +754,25 @@ func resolveRepositories(ctx context.Context, op resolveRepoOp) (repoRevisions,
 type indexedReposFunc func(ctx context.Context, revs []*search.RepositoryRevisions) (indexed, unindexed []*search.RepositoryRevisions, err error)
 type defaultReposFunc func(ctx context.Context) ([]*types.Repo, error)
 
-func defaultRepositories(ctx context.Context, getRawDefaultRepos defaultReposFunc, getIndexedRepos indexedReposFunc) ([]*types.Repo, error) {
+func defaultRepositories(ctx context.Context, getRawDefaultRepos defaultReposFunc, getIndexedRepos indexedReposFunc, excludePatterns []string) ([]*types.Repo, error) {
 	// Get the list of default repos from the db.
 	defaultRepos, err := getRawDefaultRepos(ctx)
 	if err != nil {
 		return nil, errors.Wrap(err, "querying db for default repos")
 	}
+
+	// Remove excluded repos
+	if excludePatterns != nil {
```

Comment:
```suggestion
	if len(excludePatterns) > 0 {
```

I believe the preferred style is the above, as in https://sourcegraph.com/github.com/sourcegraph/sourcegraph@8f94e730c303ea607053cbe4775f77628b62f07b/-/blob/cmd/frontend/graphqlbackend/search_results.go#L1194. `nil` slices return len 0